src/test/java/seedu/address/logic/commands/AddPatientCommandTest.java

```diff
@@ -163,7 +164,12 @@ public int getNumOfRooms() {
         }
 
         @Override
-        public Path getPathOfNumberOfRooms() {
+        public void addNumberOfRooms(int num) {
```

Comment:
is there a purpose for this method?

Reply:
It is to add the number of rooms into the hotel

Reply:
Actually why do you need to add code for room under AddPatientCommand?